code/game/machinery/Sleeper.dm

```diff
@@ -79,7 +79,8 @@
 /obj/machinery/sleeper/close_machine(mob/user)
 	if((isnull(user) || istype(user)) && state_open && !panel_open)
 		..(user)
-		if(occupant && occupant.stat != DEAD)
+		var/mob/living/mob_occupant = occupant

```

Comment:
actually, why does most of the code do isliving checks? shouldn't the typecache be enough? if admins break it, fuck admins.